Dockerfile

```diff
@@ -8,14 +8,14 @@ RUN go mod download
 
 ADD . /go/src/github.com/minio/m3/
 WORKDIR /go/src/github.com/minio/m3/
-WORKDIR /
 
-RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-w -s" -a -installsuffix cgo -o m3 ./go/src/github.com/minio/m3/
+
+RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-w -s" -a -installsuffix cgo -o m3 ./cmd/m3
```

Comment:
installsuffix is not useful https://www.reddit.com/r/golang/comments/a8uhnd/is_installsuffix_cgo_needed_with_cgo_enabled0/ - these are old stuff which is not needed.